lib/ui/views/dish_details/widgets/dish_image_header.dart

```diff
@@ -34,7 +36,7 @@ class DishImageHeader extends ViewModelWidget<DishDetailsViewModel> {
             shape: RoundedRectangleBorder(
               borderRadius: BorderRadius.circular(10),
             ),
-            color: Color(0xFFF5F5F5),
+            color: kcPopUpBg,
```

Comment:
you're using colors directly. colors should come from theme. you rmbr how to add it to theme ??

Reply:
fixed